# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1302166915 -7200
#      Thu Apr 07 11:01:55 2011 +0200
# Node ID 5329898fbe051e354f2173eeca731cf587246d76
# Parent  c3a04e23c14f3a07ef61196648c5c933aae639ed
Fixed some bugs on resizing.

diff --git a/gui//mainwindow.cpp b/gui//mainwindow.cpp
--- a/gui//mainwindow.cpp
+++ b/gui//mainwindow.cpp
@@ -22,24 +22,25 @@
 #include <QtWebKit/QWebView>
 
 #include "mainwindow.h"
 #include "octaveterminal.h"
 #include "terminal.h"
 
 MainWindow::MainWindow(QWidget *parent)
     : QMainWindow(parent) {
-    showMaximized();
     m_mdiArea = new QMdiArea();
     setCentralWidget(m_mdiArea);
 
     //addOctaveTerminal();
     loadWebPage("Online Manual", "http://www.gnu.org/software/octave/doc/interpreter/");
     addTerminalWindow();
-    m_mdiArea->setViewMode(QMdiArea::TabbedView);
+
+    m_mdiArea->setViewMode(QMdiArea::SubWindowView);
+    showMaximized();
 }
 
 MainWindow::~MainWindow() {
 
 }
 
 void MainWindow::addOctaveTerminal() {
     OctaveTerminal *octaveTerminal = new OctaveTerminal;
